sdcm/sct_events/filters.py

```diff
@@ -62,7 +63,7 @@ def msgfmt(self) -> str:
 
 class EventsFilter(BaseFilter):
     def __init__(self,
-                 event_class: Optional[Type[SctEventProtocol]] = None,
+                 event_class: Optional[Type[SctEventProtocol] | Type[StressEvent]] = None,
```

Comment:
I think, it's better to change it to `Optional[Type[SctEventProtocol] | Type[SctEvent]]`

Reply:
you mean, to remove the `= None` ?

Reply:
> you mean, to remove the `= None` ?

@enaydanov , i did this

Reply:
@fgelcer No, I just suggested the type hint, not the default value.

You need to return `None` back. It's failing unit tests

Reply:
updated